### ISSUE TITLE
Comment in dispatch of command does not hold true for all cases

### ISSUE THREAD

**Issue by Therzok** [mono/monodevelop#4918]
See https://github.com/mono/monodevelop/blob/master/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs#L1444

The above optimization does not hold true for this case: https://github.com/mono/monodevelop/blob/master/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs#L515

**Comment by slluis** [comment]
Yes, the result of `GetCommandInfo()` should be passed along to `DispatchCommand()`.

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-16] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #4925 merged by: Lluis Sanchez  
 Author: mono  
 Commit: b3a86c495bbca87fd0048ca7c1a115ae9f670f7f (xamarin/monodevelop)
